src/DemoUI/src/Services/BusinessLogic/ShopOrderService.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Packlink\DemoUI\Services\BusinessLogic;
+
+use Packlink\BusinessLogic\Http\DTO\Draft;
+use Packlink\BusinessLogic\Http\DTO\Shipment;
+use Packlink\BusinessLogic\Order\Interfaces\ShopOrderService as ShopOrderServiceInterface;
+
+/**
+ * Class OrderRepositoryService
+ *
+ * @package Packlink\PacklinkPro\Services\BusinessLogic
+ */
+class ShopOrderService implements ShopOrderServiceInterface
+{
+    /**
+     * Fetches and returns system order by its unique identifier.
+     *
+     * @param string $orderId $orderId Unique order id.
+     *
+     * @return Draft Order object.
+     *
+     */
+    public function getOrderAndShippingData($orderId)
```

Comment:
Codacy found an issue: [Avoid unused parameters such as '$orderId'.](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)

Reply:
This method is an extension point and the parameter is needed in the derived classes.